### ISSUE TITLE
Reduce power consumption at both sites

### ISSUE THREAD

**Issue by pnorman** [openstreetmap/operations#1026]
Both sites are exceeding their allowable power draw at times

<img width="621" alt="image" src="https://github.com/openstreetmap/operations/assets/1190866/e0937174-750d-4407-95f6-9c2634e5b12c">
<img width="624" alt="image" src="https://github.com/openstreetmap/operations/assets/1190866/efccf808-7db0-4cbe-aeec-903f6931409a">

Each site needs to come down in power usage by at least 250W. 10% would be better.

**Comment by tomhughes** [comment]
Have we actually been charged for any overages recently?

**Comment by pnorman** [comment]
We're at 4kVA and 3.5kVA for the new limits, I was thinking of the old ones, so we're safe.

We probably still need to reduce power consumption to meet the budget cuts the board is requiring.

**Comment by Firefishy** [comment]
AM6 base power purchased is 3.5kVA (was 3kVA a year ago)
DB4 base power purchased is 4kVA

**Comment by Firefishy** [comment]
@pnorman What do you want to do with this ticket? Maybe rephrase to match that we not exceeding budget power and say we trying to reduce the base load we buy? Or close the ticket?

**Comment by pnorman** [comment]
I'm okay closing it. We pay €4 per year per watt of power allocation. If we were to save 250W at each site, a good amount, that would be €1000. I don't feel that is worth the potential service impacts. We should look at if we want to keep the nightly tile re-rendering in local time, and if so, lower the number of cores used to spread it out over more time without impacting daytime traffic.
